google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -146,22 +148,38 @@ def _get_batch_process_metadata(
         operation_name (str):
             Required. The fully qualified operation name for a `batch_process_documents()` operation.
 
+        location (str):
+                Optional. The location of the processor used for `batch_process_documents()`.
+                Deprecated. Maintained for backwards compatibility.
+
         timeout (float):
             Optional. Default None. Time in seconds to wait for operation to complete.
             If None, will wait indefinitely.
     Returns:
         documentai.BatchProcessMetadata:
             Metadata from batch process.
     """
+    # Validate Operation Name
+    match = re.search(
+        r"projects\/\w+\/locations\/(\w+)\/operations\/\w+", operation_name
+    )
+
+    if not match:
+        raise ValueError(f"Invalid Operation Name: {operation_name}")
```

Comment:
It may help to include the expected format in the error message since it is known. `Expected operation name in the format "projects/<project>/locations/<location>/operations/<operation>"`